packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -491,10 +507,10 @@ export async function launch({
     timer.poll(blockTime);
     // This is before the initial block, we need to finish processing the
     // entire bootstrap before opening for business.
-    const runPolicy = computronCounter(params.beansPerUnit, true);
+    const runPolicy = computronCounter(params, true);
     const runSwingset = makeRunSwingset(blockHeight, runPolicy);
 
-    await runSwingset();
+    await runSwingset('queued');
```

Comment:
Why not `'forced'` here?

Reply:
Is this the "do it all in the current block and don't do anything else" one? If so, I'm ok with `forced`.

Reply:
Good point; fixed.